server/tests/maps.e2e-spec.ts

```diff
@@ -929,41 +973,104 @@ describe('Maps', () => {
     });
 
     describe('GET maps/{mapID}/credits', () => {
-        const expects = (res) => expect(res.body).toBeValidDto(MapCreditDto);
+        const expects = (res) => res.body.forEach((x) => expect(x).toBeValidDto(MapCreditDto));
 
         it('should respond with the specified maps credits', async () => {
             const res = await get(`maps/${map1.id}/credits`, 200);
-
             expects(res);
+            expect(res.body).toHaveLength(1);
         });
 
-        it('should respond with the specified maps credits with the user expand parameter', () =>
-            expandTest(`maps/${map1.id}/credits`, expects, 'user'));
-
-        // should this return a 404 instead of a 200?
-        // i agree with the ancient comment - tom
-        it('should return 200 with an empty array', async () => {
-            const res = await get('maps/999999999999999/credits', 200);
-
-            expect(res.body.mapCredits).toHaveLength(0);
+        it('should respond with the specified maps credits with the user expand parameter', async () => {
+            const res = await get(`maps/${map1.id}/credits`, 200, { expand: 'user' });
+            expects(res);
+            res.body.forEach((x) => {
+                expect(x).toHaveProperty('user');
+                expect(x.user).toBeValidDto(UserDto);
+            });
```

Comment:
`expects` won't complain if the user property isn't defined, no? Since `user` is an expand parameter, it shouldn't be defined in the test above this one which would mean `expects` would fail there, so the `forEach` here should specifically check that `user` is there.